acceptance/acceptance_test.go

```diff
@@ -1131,6 +1132,7 @@ func testAcceptance(t *testing.T, when spec.G, it spec.S, packFixturesDir, packP
 
 			packageName := createPackageLocally(aggregatePackageToml)
 			assertImageExistsLocally(packageName)
+			defer h.DockerRmi(dockerCli, nestedPackageName, packageName)
```

Comment:
We should break this into two `defer` statements. One for each respective images. The precaution is for the fact that this `defer` will not register if any of the assertions fail. We would want to `defer` the clean up as soon as the image is created.

Reply:
Changes made.